src/protocols/rdp/settings.c

```diff
@@ -269,6 +270,12 @@ enum RDP_ARGS_IDX {
      */
     IDX_SERVER_LAYOUT,
 
+    /**
+     * code_page representing the active language identifier.
+     * See [MS-RDPBCDGR] 2.2.1.11.1.1
```

Comment:
I am looking at `[MS_RDPBCDGR]` 
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/5073f4ed-1e93-45e1-b039-6e30c385867c

How do I get to `2.2.1.11.1.1` ? Is that a version or a section bookmark ?

Reply:
It's how you navigate the sections.

2. -> Messages
2.2 -> Message Flow
2.2.1 -> Connection Sequence
etc.